Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	outputtsv
	1

[Tue Sep 17 10:45:05 2024]
rule outputtsv:
    input: /home/de64/scratch/de64/2023-03-10_lDE26_Nanopore_TestRun/consensus_subsample=None.fasta, /home/de64/scratch/de64/2023-03-10_lDE26_Nanopore_TestRun/references_subsample=None.fasta, /home/de64/scratch/de64/2023-03-10_lDE26_Nanopore_TestRun/cigars_subsample=None.tsv, /home/de64/scratch/de64/2023-03-10_lDE26_Nanopore_TestRun/graph_output/inv_codebook.tsv
    output: /home/de64/scratch/de64/2023-03-10_lDE26_Nanopore_TestRun/output_subsample=None.tsv
    jobid: 0
    wildcards: subsample=None
    resources: partition=short, time=120, mem=16000, cpus=1, ntasks=1, optflags=

Traceback (most recent call last):
  File "/home/de64/marlin-nanopore/.snakemake/scripts/tmpd9980yt7.gather_data.py", line 35, in <module>
    output_data = [
  File "/home/de64/marlin-nanopore/.snakemake/scripts/tmpd9980yt7.gather_data.py", line 39, in <listcomp>
    "consensus": consdict[key],
KeyError: 0
[Tue Sep 17 10:45:13 2024]
Error in rule outputtsv:
    jobid: 0
    output: /home/de64/scratch/de64/2023-03-10_lDE26_Nanopore_TestRun/output_subsample=None.tsv

RuleException:
CalledProcessError in line 262 of /home/de64/marlin-nanopore/lDE26_Sequencing_Test.smk:
Command 'set -euo pipefail;  /home/de64/micromamba/envs/nanopore/bin/python3.10 /home/de64/marlin-nanopore/.snakemake/scripts/tmpd9980yt7.gather_data.py' returned non-zero exit status 1.
  File "/home/de64/micromamba/envs/nanopore/lib/python3.10/site-packages/snakemake/executors/__init__.py", line 2208, in run_wrapper
  File "/home/de64/marlin-nanopore/lDE26_Sequencing_Test.smk", line 262, in __rule_outputtsv
  File "/home/de64/micromamba/envs/nanopore/lib/python3.10/site-packages/snakemake/executors/__init__.py", line 551, in _callback
  File "/home/de64/micromamba/envs/nanopore/lib/python3.10/concurrent/futures/thread.py", line 58, in run
  File "/home/de64/micromamba/envs/nanopore/lib/python3.10/site-packages/snakemake/executors/__init__.py", line 537, in cached_or_run
  File "/home/de64/micromamba/envs/nanopore/lib/python3.10/site-packages/snakemake/executors/__init__.py", line 2239, in run_wrapper
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
